corehq/apps/accounting/tasks.py

```diff
@@ -15,7 +15,7 @@
 import six.moves.urllib.parse
 import six.moves.urllib.request
 from celery.schedules import crontab
-from celery.task import periodic_task, task
+from celery import shared_task
```

Comment:
Thanks for the suggestion. I also had something simpler in my mind where we just reference `shared_task` as `task` in the imports.

```
from celery import shared_task as task
```
But I wanted to keep the implementation closer to celery API. I can update the PR to use the patched code but again this would be an additional knowledge that we would have to maintain.

Reply:
I missed your comment earlier @dannyroberts, apologies for that.
I understand your concern, however, the frequency with which task files are updated would be very less. But I would raise another PR with minimal changes and rebase this PR with the master to ensure there are no conflicts.